### PR TITLE
monitoring: bump manifests to Tectonic Monitoring 1.3

### PR DESCRIPTION
**Pull request by brancz**
This release of Tectonic Monitoring introduces:

* Cluster Alerting (a collection of pre-built alerting rules)
* A 2 node Alertmanager cluster
* Configurable container image repos and configurable resources of the Tectonic Monitoring components (Alertmanager & Prometheus)
* Discovering more Kubernetes components

Tested this with the experimental flag on.

@s-urbaniak @alexsomesan 

/cc @robszumski @rezaloo Docs still are to be done for this, but will be written in the tectonic-prometheus-operator repo.

/cc @colhom this coincidentally fixes all monitoring related parts of #779

### CODE REVIEW

File: modules/tectonic/resources/manifests/monitoring/alertmanager.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: Alertmanager
+metadata:
+  name: main
+  namespace: tectonic-system
+  labels:
+    k8s-app: alertmanager
+    alertmanager: main
+spec:
+  replicas: 2
```
Comment (s-urbaniak) [inline modules/tectonic/resources/manifests/monitoring/alertmanager.yaml]: does it make to configure this setting?

Comment (brancz) [inline modules/tectonic/resources/manifests/monitoring/alertmanager.yaml]: At some point yes, but for introducing this it's fine, plus I wanted to keep the templating effort to a minimum. I'd expect this only to become configurable once the experimental flag disappears.

**Comment by s-urbaniak** [comment]
ok to test

**Comment by s-urbaniak** [comment]
retest this please

**Comment by s-urbaniak** [comment]
tested and verified, that monitoring graphs appear, merging!

**Pull request merged**